lib/features/manual/widget/showcase_text_style.dart

```diff
@@ -13,7 +13,7 @@ class ShowcaseTextStyle extends StatelessWidget {
     return Text(
       descriptionText,
       style: const TextStyle(
-        color: Colors.white,
+        color: Colors.black,
```

Comment:
no need to set the color, the default for the brightness of the app will work fine with surface color